docker/images/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.2.4
+
+USER 1001
```

Comment:
Why is this required if OpenShift already changes the default user when running this image?

Reply:
This helps openshift detect that the container isn't running as root. Not sure entirely why this is needed but this make the image usable in openshift.

1001 is just a random choice, could be anything else.